Source/DafnyLanguageServer/Handlers/Custom/DafnyCounterExampleHandler.cs

```diff
@@ -16,16 +16,27 @@ namespace Microsoft.Dafny.LanguageServer.Handlers.Custom {
   public class DafnyCounterExampleHandler : ICounterExampleHandler {
     private readonly ILogger logger;
     private readonly IDocumentDatabase documents;
+    private readonly ITextDocumentLoader documentLoader;
 
-    public DafnyCounterExampleHandler(ILogger<DafnyCounterExampleHandler> logger, IDocumentDatabase documents) {
+    public DafnyCounterExampleHandler(ILogger<DafnyCounterExampleHandler> logger, ITextDocumentLoader documentLoader, IDocumentDatabase documents) {
       this.logger = logger;
       this.documents = documents;
+      this.documentLoader = documentLoader;
     }
 
     public async Task<CounterExampleList> Handle(CounterExampleParams request, CancellationToken cancellationToken) {
       try {
         var document = await documents.GetLastDocumentAsync(request.TextDocument);
+
         if (document != null) {
+          var verificationTasks = document!.VerificationTasks;
```

Comment:
You can save one level of nesting with something like `if (document?.VerificationTasks is {} vts) {` or `if (document is { VerificationTasks: {} vts })`?

Reply:
It's 
```c#
if (A) 
{ 
  if (B) { 
    C() 
  } 
  D() 
}
```
How would I get rid of one `if` ?

Reply:
Woops, thanks